config/server.properties

```diff
@@ -11,3 +11,4 @@
 #
 
 host.name=localhost
+clustermap.cluster.name=Ambry_Dev
```

Comment:
for dev setup, we don't currently handle Helix based instantiation. This can be added later. For dev testing, we need to use the configs named with 'helix' in them. Will add a write-up at some point on the steps.